src/modules/sudo.rs

```diff
@@ -18,7 +18,15 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
         return None;
     }
 
-    let is_sudo_cached = context.exec_cmd("sudo", &["-n", "true"]).is_some();
+    let mut sudo_option = "-nv";
```

Comment:
We don't need to have a `mut` here. Instead, you could do something like

```
let has_N_option = context.exec_cmd("sudo", &["-NV"]).is_some();
let sudo_option = if has_N_option {
  "-Nnv"
} else {
  "-nv"
}
```

You could even fold the `has_N_option` directly into the `if` statement, though I'm usually a little bit wary about conditionals with potential side effects.